.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: groovy
```

Comment:
I'm honestly not sure... this was a long time ago. I've changed it to `java` as I agree that makes more sense.